Tasks/PublishBuildArtifactsV1/task.json

```diff
@@ -75,6 +75,15 @@
             "required": false,
             "helpMarkDown": "Enter the degree of parallelism, or number of threads used, to perform the copy. The value must be at least 1 and not greater than 128.",
             "visibleRule": "ArtifactType = FilePath && Parallel = true"
+        },
+        {
+            "name": "FileCopyOptions",
```

Comment:
Should we put this input under an "Advanced" group?

Reply:
Good idea! I've added an Advanced group and put this option there. There are some other options that could fit there as well, but I didn't want to move around existing options and confuse users.